torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -66,11 +69,11 @@ def fn(
 
     def _parse_fill(
         self, image: Union[PIL.Image.Image, torch.Tensor, features.Image], num_channels: int
-    ) -> Optional[List[float]]:
+    ) -> Union[int, float, Sequence[int], Sequence[float]]:
         fill = self.fill
 
-        if isinstance(image, PIL.Image.Image) or fill is None:
-            return fill
+        # if isinstance(image, PIL.Image.Image) or fill is None:
```

Comment:
Nit: seems like we don't need `image` any more here.

Reply:
Yes, I'll remove this code once we agreed on `fill=None` or `fill=0`. Probably, if we go deprecation cycle I had to get this code back